promptsource/promptsource.py

```diff
@@ -54,14 +54,15 @@ def reset_template_state():
 
 
 def show_jinja(t, width=WIDTH):
-    wrap = textwrap.fill(t, width=width, replace_whitespace=False)
-    out = highlight(wrap, DjangoLexer(), HtmlFormatter())
+    wrap = [textwrap.fill(subt, width=width, replace_whitespace=False) for subt in t.split("\n")]
+    wrap = "\n".join(wrap)
```

Comment:
Double newlines sometimes got ignored (by textwrap?) so we have to trick it.

Reply:
```suggestion
    wrap = textwrap.fill(t, width=width, replace_whitespace=False, drop_whitespace=False)
```

This is probably equivalent but nicer.

Reply:
No, I was wrong. They are not the same and the special treatment still works better than `drop_whitespace=False`,